keyboards/mechlovin/th1800/rules.mk

```
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32a
+
+# Processor frequency
+F_CPU = 16000000
+
+# Bootloader selection
+BOOTLOADER = bootloadHID
```

```suggestion
BOOTLOADER = USBasp
```
If the board is using USBasploader as the `readme` mentions.